drupal/web/modules/custom/covid_ui/covid_ui.routing.yml

```diff
@@ -12,15 +12,15 @@ covid_ui.translation:
     _form: '\Drupal\covid_ui\Form\TranslateForm'
     _title: 'Přeložit'
   requirements:
-    _permission: 'translate any entity'
+    _permission: 'translate interface'
```

Comment:
preco toto obmedzujes? ved ti translatori to budu chciet prelozit ci nie?

Reply:
Nooo ... tady je trochu problém že by to z toho mohl kdokoliv mazat a jestli to jsou relativně omezené role, trochu bych se bál :/